Documentation/coding-guidelines/breaking-changes.md

```diff
@@ -16,47 +22,86 @@ To help triage breaking changes, we classify them in to four buckets:
 3. Unlikely Grey Area
 4. Clearly Non-Public
 
-### Bucket 1: Public Contract
+## Bucket 1: Public Contract
 *Clear violation of public contract.*
 
 Examples:
-* throwing a new/different exception type in an existing common scenario
-* An exception is no longer thrown
-* A different behavior is observed after the change for an input
 * renaming a public type, member, or parameter
-* decreasing the range of accepted values within a given parameter
-* A new instance field is added to a type (impacts serialization)
 * changing the value of a public constant or enum member
```

Comment:
❗️ I think it's important to include binary breaking changes in this section (excluding potentially-avoidable ones like serialization issues), even if it's just a link to an explanation elsewhere.